tests/Microsoft.Bot.Builder.AI.QnA.Tests/QnAMakerCardEqualityComparer.cs

```diff
@@ -30,6 +30,12 @@ public bool Equals(IActivity x, IActivity y)
                 return false;
             }
 
+            if ((activity1.Attachments == null && activity2.Attachments != null) ||
```

Comment:
Great test. However this statement is really hard to follow. simplify.

Reply:
Thanks. Updated changes.

---
In reply to: [430562693](https://github.com/microsoft/botbuilder-dotnet/pull/3935#discussion_r430562693) [](ancestors = 430562693)